go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
 	github.com/openshift/cluster-network-operator v0.0.0-20190207145423-c226dcab667e // indirect
 	github.com/openshift/hive v0.0.0-20200210203046-8b1c393442db
-	github.com/openshift/operator-custom-metrics v0.3.1-0.20200901174648-463079905232
+	github.com/openshift/operator-custom-metrics v0.4.0
```

Comment:
This seems like a separate, unrelated change.

Reply:
But without it the build failed because that commit is now it's own tag. Do you want it in a separate PR?

Reply:
Sigh, I see.

Yeah, it really should go in a separate PR, which I guess needs to be in front of this one.

Reply:
It builds without this change as well. I'll create a separate PR anyway.